front/app/components/PostShowComponents/Comments/ParentComment.tsx

```diff
@@ -81,12 +79,12 @@ const ParentComment = ({
     const projectId = idea?.data.relationships.project.data.id || null;
     const commentDeleted =
       comment.data.attributes.publication_status === 'deleted';
-    const commentingEnabled =
+    const commentingDisabledReason =
       postType === 'initiative'
-        ? commentingPermissionInitiative?.enabled === true
-        : idea?.data.attributes.action_descriptor.commenting_idea.enabled ===
-          true;
-    const showCommentForm = authUser && commentingEnabled && !commentDeleted;
+        ? commentingPermissionInitiative?.disabledReason
+        : idea?.data.attributes.action_descriptor.commenting_idea
+            .disabled_reason;
+    const showCommentForm = isNil(commentingDisabledReason) && !commentDeleted;
```

Comment:
This does the same as authUser + checking if commenting is enabled. If it's disabled, or the user is not logged, there will be a commenting disabled reason. So I think it's a more robust check.